integration_tests/test_basic.py

```diff
@@ -562,8 +524,61 @@ def test_batch_tx(cronos):
     txs = [
         w3.eth.get_transaction_by_block(receipts[0].blockNumber, i) for i in range(3)
     ]
-    for tx, signed in zip(txs, signed_txs):
-        assert tx.hash == signed.hash
+    for tx, h in zip(txs, tx_hashes):
+        assert tx.hash == h
+
+
+def test_failed_transfer_tx(cronos):
+    """
+    It's possible to include a failed transfer transaction in batch tx
+    """
+    w3 = cronos.w3
+    cli = cronos.cosmos_cli()
+    sender = ADDRS["community"]
+    recipient = ADDRS["validator"]
+    nonce = w3.eth.get_transaction_count(sender)
+    half_balance = w3.eth.get_balance(sender) // 3 + 1
+
+    # build batch tx, the third tx will fail, but will be included in block
+    # because of the batch tx.
+    transfer1 = {"from": sender, "nonce": nonce, "to": recipient, "value": half_balance}
+    transfer2 = {
+        "from": sender,
+        "nonce": nonce + 1,
+        "to": recipient,
+        "value": half_balance,
+    }
+    transfer3 = {
+        "from": sender,
+        "nonce": nonce + 2,
+        "to": recipient,
+        "value": half_balance,
+    }
+    cosmos_tx, tx_hashes = build_batch_tx(
+        w3, cli, [transfer1, transfer2, transfer3], KEYS["community"]
+    )
+    rsp = cli.broadcast_tx_json(cosmos_tx)
+    assert rsp["code"] == 0, rsp["raw_log"]
+
+    receipts = [w3.eth.wait_for_transaction_receipt(h) for h in tx_hashes]
+    assert receipts[0].status == receipts[1].status == 1
+    assert receipts[2].status == 0
+
+    # test the cronos_getTransactionReceiptsByBlock api
+    rsp = get_receipts_by_block(w3, receipts[0].blockNumber)
```

Comment:
the new api is called here, and the result is checked against the normal `eth_getTransactionReceipt` response.